python.md

```diff
@@ -3,118 +3,143 @@ title: Python
 category: Python
 ---
 
-### Tuples (immutable)
+### Tuples (Immutable)
 
 ```py
-tuple = ()
+t = ()  # Empty tuple
+t = (1, 2, 3)  # Tuple with elements
+t = (1,)  # Single-element tuple (comma is required)
+
+# Accessing elements
+t[0]  # First element
+t[-1]  # Last element
+
+# Methods
+len(t)  # Get the length
 ```
 
-### Lists (mutable)
+### Lists (Mutable)
 
 ```py
-list = []
-list[i:j]  # returns list subset
-list[-1]   # returns last element
-list[:-1]  # returns all but the last element
-*list      # expands all elements in place
+lst = []  # Empty list
+lst = [1, 2, 3]  # List with elements
+
+# Accessing and slicing
+lst[i:j]  # Sublist from index i to j-1
+lst[-1]  # Last element
+lst[:-1]  # All but the last element
+
+# Modifications
+lst[i] = value  # Set an element
+lst.append(value)  # Add to the end
+lst.insert(i, value)  # Insert at position i
+lst.extend(other)  # Add multiple elements
+lst.pop(i=-1)  # Remove and return element (default: last)
+lst.remove(value)  # Remove first occurrence
+del lst[i:j]  # Delete slice
```

Comment:
```suggestion
```suggestion
lst.extend([1,2,3])  # Extends list with another list
lst.pop()  # Remove and return element. Can also accept an index.
lst.remove(value)  # Remove first occurrence of value
del lst[i:j]  # Delete slice between i and j-1
```